SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Reporting Security Issues
+
+Please report security issues **confidentially** using
+[GitHub's form](https://github.com/SBOMit/specification/security/advisories/new).
+Alternatively, you can send an encrypted email to `jcappos@nyu.edu` using the
```

Comment:
Should remove use of email to any single person which may not generally allow for a timely response by the group as a whole.

Reply:
Okay, I've removed that aspect.